tgui/packages/tgui/interfaces/Smes.js

```diff
@@ -0,0 +1,188 @@
+                    fillValue={inputAvailable / POWER_MUL}
+                    minValue={0}
+                    maxValue={inputLevelMax / POWER_MUL}
+                    step={5}
+                    stepPixelSize={4}
+                    format={value => formatPower(value * POWER_MUL, 1)}
+                    onDrag={(e, value) => act('input', {
+                      target: value * POWER_MUL,
+                    })} />
+                </Flex.Item>
+                <Flex.Item>
+                  <Button
+                    icon="forward"
+                    disabled={inputLevel === inputLevelMax}
+                    onClick={() => act('input', {
+                      adjust: 10000,
+                    })} />
+                  <Button
+                    icon="fast-forward"
+                    disabled={inputLevel === inputLevelMax}
+                    onClick={() => act('input', {
+                      target: 'max',
+                    })} />
+                </Flex.Item>
+              </Flex>
+            </LabeledList.Item>
+            <LabeledList.Item label="Available">
+              {formatPower(inputAvailable)}
+            </LabeledList.Item>
+          </LabeledList>
+        </Section>
+        <Section title="Output">
+          <LabeledList>
+            <LabeledList.Item
+              label="Output Mode"
+              buttons={
+                <Button
+                  icon={outputAttempt ? 'power-off' : 'times'}
+                  selected={outputAttempt}
+                  onClick={() => act('tryoutput')}>
+                  {outputAttempt ? 'On' : 'Off'}
+                </Button>
+              }>
+              <Box color={outputState}>
+                {outputting
+                  ? 'Sending'
+                  : charge > 0
+                    ? 'Not Sending'
+                    : 'No Charge'}
+              </Box>
+            </LabeledList.Item>
+            <LabeledList.Item label="Target Output">
+              <Flex inline width="100%">
+                <Flex.Item>
+                  <Button
+                    icon="fast-backward"
+                    disabled={outputLevel === 0}
+                    onClick={() => act('output', {
+                      target: 'min',
+                    })} />
+                  <Button
+                    icon="backward"
+                    disabled={outputLevel === 0}
+                    onClick={() => act('output', {
+                      adjust: -10000,
+                    })} />
+                </Flex.Item>
+                <Flex.Item grow={1} mx={1}>
+                  <Slider
+                    value={outputLevel / POWER_MUL}
+                    minValue={0}
+                    maxValue={outputLevelMax / POWER_MUL}
+                    step={5}
+                    stepPixelSize={4}
+                    format={value => formatPower(value * POWER_MUL, 1)}
+                    onDrag={(e, value) => act('output', {
```

Comment:
`onChange` here so people cant overload the server with mass hrefs by dragging the slider up and down without letting go.